homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/exeption/UserValidationService.java

```diff
@@ -0,0 +1,28 @@
+package com.tutrit.java.quickstart.homework.exeption;
+
+public class UserValidationService {
+
```

Comment:
Best practice obviously specify modificator  access for methods.
Also I recomend you read about incapsulation.